lib/tasks_private/lib/simple_logging.rb

```diff
@@ -0,0 +1,12 @@
+# frozen_string_literal: true
+
+require 'logger'
+require 'active_support/core_ext/module/delegation'
+
+module Kernel
+  private
+  def logger
+    @logger ||= Logger.new(STDOUT)
+  end
+  delegate *%i(info debug warn error fatal), :to => :logger
+end
```

Comment:
Why do you need this?  if all you are trying to do is call a logger, just set a global `$log = Logger.new(STDOUT)`, and be done with it.  I'd prefer that over patching Kernel.